Automattic/woocommerce-payments
# Fix Afterpay logo size on settings page

Fixes #8324

#### Changes proposed in this Pull Request

This PR fixes the size of the Afterpay logo on settings page.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/2c2d7e2d-1052-437d-96e7-e27af946ccfc)

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/888c4c1f-6aa3-42c6-89bb-8e7ee3254a11)


#### Testing instructions





* Open the settings page and ensure the Afterpay logo is 64x40 and the margin is correct

-------------------


**Post merge**
